### PR TITLE
Adding Related Applications sample for #132

### PR DESCRIPTION
**Pull request by PaulKinlan**

### CODE REVIEW

**Comment by PaulKinlan** [comment]
@jeffposnick or @addyosmani would love a review. ta.

File: app-install-banner/related-applications/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+limitations under the License.
+-->
+
+<html>
+  <head>
+    <title>Web App Install Banner: Related Apps (Android currently) Sample</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/samples/styles/main.css">
+    <link rel="manifest" href="manifest.json">
+    <script>
```
Comment (ebidel) [inline app-install-banner/related-applications/index.html]: scripts at the bottom yo

File: app-install-banner/related-applications/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    <p>The <a href="http://updates.html5rocks.com/2014/11/Support-for-installable-web-apps-with-webapp-manifest-in-chrome-38-for-Android">related (Native) app install banner</a> will prompt the user to add your web app to the users device.
+    It will only prompt when a number of criteria have been met:
+    </p>
+
+    <ul>
+      <li>The site is using HTTPS</li>
+      <li>The site has a manifest declared</li>
+      <li>The manifest has a <code>short_name</code>, 144 pixel icon and a type of 'image/png'</li>
+      <li>The manifest has a <code>related_applications</code> section with an object that has they 'type': 'play' and a Store ID.</li>
```
Comment (ebidel) [inline app-install-banner/related-applications/index.html]: they -> the

Comment (ebidel) [inline app-install-banner/related-applications/index.html]: Store -> Google Play Store

Comment (ebidel) [inline app-install-banner/related-applications/index.html]: It looks like a package name, right?

**Comment by ebidel** [comment]
Reviewed Pk!

**Comment by PaulKinlan** [comment]
@ebidel dude missed this.

File: app-install-banner/related-applications/index.html (hunk 1 above)
Comment (PaulKinlan) [inline app-install-banner/related-applications/index.html]: All the others don't...

**Comment by PaulKinlan** [comment]
@ebidel could you have a quick review and pull... ta bud.

File: app-install-banner/related-applications/readme.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+App Install Banner Sample
+===
+
+See https://googlechrome.github.io/samples/app-install-banner/basic-banner/index.html for a live demo.
```
Comment (jeffposnick) [inline app-install-banner/related-applications/readme.md]: basic-banner → related-applications

**Comment by jeffposnick** [comment]
Feel free to merge after resolving my one comment.

**Comment by PaulKinlan** [comment]
Good catch.

**Pull request merged**